deepspeed/launcher/runner.py

```diff
@@ -193,6 +193,8 @@ def parse_args(args=None):
                         "numbers and range. i.e. 1,3-5,7 => [1,3,4,5,7].  When not "
                         "specified, all cores on system would be used rank binding")
 
+    parser.add_argument("--port", type=int, default=22, help="SSH port to use for remote connections")
```

Comment:
Can we change this flag to be `--ssh_port` instead? I just want to make it clear which port we are defining here, since we also have `--master_port`.

Reply:
Apologies for the confusing name. Fixed as requested.